ncl/ncl_index/ncl_index.rst

```diff
@@ -13,3 +13,5 @@ NCL Index
 
 .. note::
     For more information on how we verify these recommendations, see the :ref:`receipts`.
+
+NCL_spectral_analysis
```

Comment:
Is this meant to be here?

Reply:
For `NCL_spectral_analysis.ipynb` to show up on the sidebar for `NCL Applications` is can be added to be `ncl_entries.rst` file